packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -325,7 +325,10 @@ export class LedgerHDWallet implements core.HDWallet, core.BTCWallet, core.ETHWa
   public async getFeatures (): Promise<any> {
     const res = await this.transport.getDeviceInfo()
     handleError(res, this.transport)
```

Comment:
The `res.payload` stuff, and the `handleError()` call are both assuming that the `getDeviceInfo()` call went through `this.transport.call()`. It would be better if that end was fixed, so then we get these messages and the responses when logging from `keyring.onAny(...)`, rather than changing this end of it.